packages/integrations/useFocusTrap/demo.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import { ref } from 'vue-demi'
+import { useFocusTrap } from '.'
+
+const target = ref()
+const { hasFocus, activate, deactivate } = useFocusTrap(target)
+</script>
+
+<template>
+  <div class="flex flex-col items-center">
+    <button @click="activate()">
+      Trap Focus
+    </button>
+    <input
+      type="text"
+      :placeholder="hasFocus ? 'You can\'t foucs me' : 'You can focus me'"
```

Comment:
The demo looks interesting 👍

Maybe a line of "The focus is trapped in the form" somewhere could better showing the feature?